examples/java/CMakeLists.txt

```diff
@@ -32,7 +33,7 @@ endif()
 
 foreach (example app-bittorrent app-centralizedmutex app-masterworker app-pingpong app-tokenring async-yield async-waitall async-dsend
          cloud-migration cloud-masterworker dht-chord dht-kademlia energy-consumption energy-pstate energy-vm io-file io-storage
-         process-kill process-migration process-startkilltime process-suspend task-priority trace-pingpong)
+         process-kill process-migration process-startkilltime process-suspend task-priority trace-pingpong hostload)
```

Comment:
please keep this list in alphabetical order

Reply:
done